pkg/streamingpromql/operators/aggregation.go

```diff
@@ -46,9 +47,12 @@ type group struct {
 	// Used to sort groups in the order that they'll be completed in.
 	lastSeriesIndex int
 
-	// Sum and presence for each step.
-	sums    []float64
-	present []bool
+	// Sum, presence, and histograms for each step.
+	floatSums           []float64
+	floatPresent        []bool
+	floatPointCount     int
+	histogramValues     []*histogram.FloatHistogram
```

Comment:
[nit] Not sure the `values` is adding much to this name

```suggestion
	histogram     []*histogram.FloatHistogram
```